### ISSUE TITLE
Resolve homepage React error

### ISSUE THREAD

**Issue by kyranjamie** [blockstack/blockstack-app#3634]
<img width="1288" alt="image" src="https://user-images.githubusercontent.com/1618764/235639082-413a516a-172a-4a0a-9065-c1239df60883.png">

<details>
<summary>error</summary>

```
act-dom.development.js:86 Warning: React does not recognize the `hasCopied` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `hascopied` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    at div
    at eval (webpack-internal:///./node_modules/@emotion/react/dist/emotion-element-6a883da9.browser.esm.js:57:66)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/box/index.esm.js:17:22)
    at eval (webpack-internal:///./src/app/components/crypto-assets/crypto-currency-asset/crypto-currency-asset-item.layout.tsx:39:5)
    at eval (webpack-internal:///./src/app/components/crypto-assets/crypto-currency-asset/crypto-currency-asset-item.tsx:29:5)
    at div
    at eval (webpack-internal:///./node_modules/@emotion/react/dist/emotion-element-6a883da9.browser.esm.js:57:66)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/box/index.esm.js:17:22)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/stack.esm.js:27:24)
    at BalancesList (webpack-internal:///./src/app/features/balances-list/balances-list.tsx:38:3)
    at div
    at eval (webpack-internal:///./node_modules/@emotion/react/dist/emotion-element-6a883da9.browser.esm.js:57:66)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/box/index.esm.js:17:22)
    at Transition (webpack-internal:///./node_modules/react-transition-group/esm/Transition.js:133:30)
    at Transition (webpack-internal:///./node_modules/@stacks/ui/dist/transition/base.esm.js:14:21)
    at SlideFade (webpack-internal:///./node_modules/@stacks/ui/dist/transition/slide-fade.esm.js:31:36)
    at Suspense
    at div
    at eval (webpack-internal:///./node_modules/@emotion/react/dist/emotion-element-6a883da9.browser.esm.js:57:66)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/box/index.esm.js:17:22)
    at div
    at eval (webpack-internal:///./node_modules/@emotion/react/dist/emotion-element-6a883da9.browser.esm.js:57:66)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/box/index.esm.js:17:22)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/stack.esm.js:27:24)
    at HomeTabs (webpack-internal:///./src/app/pages/home/components/home-tabs.tsx:32:5)
    at div
    at eval (webpack-internal:///./node_modules/@emotion/react/dist/emotion-element-6a883da9.browser.esm.js:57:66)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/box/index.esm.js:17:22)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/stack.esm.js:27:24)
    at div
    at eval (webpack-internal:///./node_modules/@emotion/react/dist/emotion-element-6a883da9.browser.esm.js:57:66)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/box/index.esm.js:17:22)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/stack.esm.js:27:24)
    at HomeLayout (webpack-internal:///./src/app/pages/home/components/home.layout.tsx:22:5)
    at HomeContainer (webpack-internal:///./src/app/pages/home/home.tsx:54:3)
    at HomeLoader (webpack-internal:///./src/app/pages/home/home.loader.tsx:16:3)
    at Home
    at AccountGate (webpack-internal:///./src/app/routes/account-gate.tsx:29:3)
    at RenderedRoute (webpack-internal:///./node_modules/react-router/dist/index.js:715:5)
    at Outlet (webpack-internal:///./node_modules/react-router/dist/index.js:1118:26)
    at div
    at eval (webpack-internal:///./node_modules/@emotion/react/dist/emotion-element-6a883da9.browser.esm.js:57:66)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/box/index.esm.js:17:22)
    at div
    at eval (webpack-internal:///./node_modules/@emotion/react/dist/emotion-element-6a883da9.browser.esm.js:57:66)
    at eval (webpack-internal:///./node_modules/@stacks/ui/dist/box/index.esm.js:17:22)
    at ContainerLayout (webpack-internal:///./src/app/features/container/container.layout.tsx:16:5)
    at Container (webpack-internal:///./src/app/features/container/container.tsx:23:100)
    at RenderedRoute (webpack-internal:///./node_modules/react-router/dist/index.js:715:5)
    at Routes (webpack-internal:///./node_modules/react-router/dist/index.js:1209:5)
    at AppRoutesAfterUserHasConsented (webpack-internal:///./src/app/routes/app-routes.tsx:127:69)
    at AppRoutes (webpack-internal:///./src/app/routes/app-routes.tsx:348:96)
    at ErrorBoundary (webpack-internal:///./src/app/features/errors/error-boundary.tsx:20:1)
    at AppErrorBoundary (webpack-internal:///./src/app/features/errors/app-error-boundary.tsx:86:3)
    at Router (webpack-internal:///./node_modules/react-router/dist/index.js:1141:15)
    at HashRouter (webpack-internal:///./node_modules/react-router-dom/dist/index.js:372:5)
    at Suspense
    at ColorModeProvider (webpack-internal:///./node_modules/@stacks/ui/dist/color-modes/index.esm.js:35:26)
    at QueryClientProvider (webpack-internal:///./node_modules/@tanstack/react-query/build/lib/QueryClientProvider.mjs:47:3)
    at ThemeSwitcherProvider (webpack-internal:///./src/app/common/theme-provider.tsx:49:3)
    at ThemeProvider (webpack-internal:///./node_modules/@emotion/react/dist/emotion-element-6a883da9.browser.esm.js:96:64)
    at ThemeProvider (webpack-internal:///./node_modules/@stacks/ui/dist/theme-provider.esm.js:20:25)
    at PersistGate (webpack-internal:///./node_modules/redux-persist/es/integration/react.js:39:5)
    at Provider (webpack-internal:///./node_modules/react-redux/es/components/Provider.js:16:3)
    at App
p
```

</details>

**Comment by markmhendrickson** [comment]
Any particular context in which this appears?

**Comment by kyranjamie** [comment]
@markmhx Just landing on the homepage

**Comment by markmhendrickson** [comment]
I mean, is there any particular wallet state when trying to load the homepage and getting this? It's hard to tell atm how prevalent it might be or what might be causing it generally

**Comment by kyranjamie** [comment]
> I mean, is there any particular wallet state when trying to load the homepage and getting this? It's hard to tell atm how prevalent it might be or what might be causing it generally

This doesn't cause any user-facing error, or even an error in production. It's just annoying/bad practice to leave errors running.

@alter-eggo has fixed it already

**Comment by kyranjamie** [comment]
Closed by #3643
